pkgs/development/haskell-modules/lib/default.nix

```
@@ -354,4 +354,32 @@ rec {
   # same package in the (recursive) dependencies of the package being
   # built. Will delay failures, if any, to compile time.
   allowInconsistentDependencies = compose.allowInconsistentDependencies;
+
+
+  # Collect all the cabal2nix derivers (i.e. nix file generated from a call
+  # to cabal2nix) used in a list of packages and save them as a link farm.
```

```suggestion
  # to cabal2nix) used in a list of packages and merge them together as one package containing links to all of them.
```
Maybe like that? Not sure

---

I removed that part of the changes, so this is no longer relevant: I'm checking "Resolved".